integration-tests/test_client_options.py

```diff
@@ -104,21 +103,14 @@ def test_group(insights_client, tmp_path):
             and value
     """
     group_name = "testing-group"
-    archive_name = "archive.tar.gz"
-    archive_location = tmp_path / archive_name
 
     # Running insights-client in offline mode to generate archive
     insights_client.run(
-        "--offline", f"--group={group_name}", f"--output-file={archive_location}"
+        "--offline", f"--group={group_name}", f"--output-dir={tmp_path}"
     )
 
-    with tarfile.open(archive_location, "r") as tar:
-        dir_name = tar.getnames()[0]
-        for m in tar.getmembers():
-            if m.name == f"{dir_name}/tags.json":
-                tag_file = tar.extractfile(m)
-                tag_file_content = json.load(tag_file)
-                break
+    with open(f"{tmp_path}/data/tags.json", "r") as f:
```

Comment:
since this is a `pathlib.Path` now, let's use that way:
```python
with (tmp_path / "data" / "tags.json").open() as f:
   ...
```
alternatively, you can load the file at once:
```python
  ... = json.loads(that_path.read_text())
```